docs/rabbitmq.config.example

```diff
@@ -338,6 +372,26 @@
    %%
    %% {queue_index_embed_msgs_below, 4096},
 
+   %% The credits that a queue process is given by the message store.
+   %%
+   %% By default, a queue process is given 4000 message store credits,
+   %% and then 800 for every 800 messages that it processes.
+   %%
+   %% A queue process persists AMQP messages by sending them 1 by 1 to the message store process.
+   %% If the queue process sent 4000 AMQP messages and the message store process didn't manage to persist 800,
```

Comment:
It does not have to persist messages, just process them. Credit flow ack is sent before message store handles a message, it will take some time to actually persist messages.